internal/providers/terraform/azure/testdata/storage_table_test/storage_table_test.tf

```diff
@@ -2,21 +2,85 @@ provider "azurerm" {
   features {}
 }
 
+locals {
+  account_replication_types = ["LRS", "ZRS", "GRS", "RA-GRS", "GZRS", "RA-GZRS"]
```

Comment:
I listed all these types here so we could `for_each` through them in the tests to create tests for every one of them.